packages/rum-core/src/domain/configuration/remoteConfiguration.ts

```diff
@@ -0,0 +1,53 @@
+import type { DefaultPrivacyLevel } from '@datadog/browser-core'
+import { display, addEventListener, assign } from '@datadog/browser-core'
+import type { RumInitConfiguration } from './configuration'
+
+export const REMOTE_CONFIGURATION_URL = 'https://www.datad0g-browser-agent.com/configuration'
```

Comment:
For testing purposes, I hosted the RC in the same CDN as the bundles but it will change once the Remote config provides their CDN